nemo/collections/asr/parts/submodules/ctc_decoding.py

```diff
@@ -247,6 +253,13 @@ def __init__(self, decoding_cfg, blank_id: int):
             self.compute_timestamps |= self.preserve_frame_confidence
 
         if self.cfg.strategy == 'greedy':
+            if self.cfg.get('beam', None):
```

Comment:
Dont log this warning. strategy=beam or greedy decides what config gets used

Reply:
done